crates/bindings-csharp/Codegen/Module.cs

```diff
@@ -19,7 +19,7 @@ enum ColumnAttrs : byte
 
 record ColumnDeclaration : MemberDeclaration
 {
-    public readonly ColumnAttrs Attrs;
+    public readonly EquatableArray<(string?, ColumnAttrs)> Attrs;
```

Comment:
Can you use named tuple fields, or better yet, extract this to a `record struct`? All the `.Item1` / `.Item2` tend to be not very clear when reading.